iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/ArgInstructionImplTest.java

```diff
@@ -58,11 +59,9 @@ void argInstructionSimple() {
     assertThat(tree.children()).hasSize(2);
     assertThat(tree.keyValuePairs()).hasSize(1);
 
-    KeyValuePair keyValuePair = tree.keyValuePairs().get(0);
+    NewKeyValuePair keyValuePair = tree.keyValuePairs().get(0);
     assertThat(keyValuePair.getKind()).isEqualTo(DockerTree.Kind.KEY_VALUE_PAIR);
-    assertThat(keyValuePair.key().value()).isEqualTo("key1");
-    assertThat(keyValuePair.equals()).isNull();
-    assertThat(keyValuePair.value()).isNull();
```

Comment:
We globally lose the check on the `equals` operator.
Not necessarily a big miss.

Reply:
Maybe we will have the rule to check for the equal sign. However, I think it is not important for now.